doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -122,6 +122,13 @@ Declaring Coercions
   and then declares :token:`ident` as a coercion between it source and its target.
   Both forms support the :attr:`local` attribute, which makes the coercion local to the current section.
 
+  This command supports the :attr:`local` and :attr:`global` attributes.
+  They are described :ref:`here <set_unset_scope_qualifiers>`.
+
+  .. attr:: nonuniform
+
+     Silence the non uniform inheritance warning (see below).
```

Comment:
```suggestion
     Silence the non uniform inheritance warning.
```

I think "see below" is not helpful here because the warning message just repeats what's said here.
Furthermore the target is half a page below and a bit buried.